python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -1015,6 +1015,11 @@ class BuiltinDataset(OnDiskDataset):
     _all_datasets = _datasets + _large_datasets
 
     def __init__(self, name: str, root: str = "datasets") -> OnDiskDataset:
+        # For user using DGL 2.2 or later version, we prefer them to use
+        # datasets with `seeds` suffix. This hack should be removed, when the
+        # datasets with `seed` suffix have covered previous ones.
+        if "seeds" not in name:
+            name += "-seeds"
```

Comment:
all existing BuiltinDataset has the `-seeds` counterpart for now? even `ogb-lsc-mag240m`? This change will apply to all existing datasets.

Reply:
Yes, I have added seeds version for all existing BuiltinDataset previously.